src/main/java/com/exasol/releasedroid/maven/MavenPlatformValidator.java

```diff
@@ -18,13 +16,18 @@
     private static final List<String> REQUIRED_PLUGINS = List.of("nexus-staging-maven-plugin", "maven-source-plugin",
             "maven-gpg-plugin", "maven-javadoc-plugin", "maven-deploy-plugin");
     protected static final String MAVEN_WORKFLOW_PATH = ".github/workflows/maven_central_release.yml";
+    private final JavaRepository repository;
+
+    public MavenPlatformValidator(final JavaRepository repository) {
+        this.repository = repository;
+    }
 
     @Override
     // [impl->dsn~validate-maven-release-workflow-exists~1]
-    public Report validate(final Repository repository) {
+    public Report validate() {
         final Report report = Report.validationReport();
-        report.merge(validateFileExists(repository, MAVEN_WORKFLOW_PATH, "Workflow for a Maven release."));
-        report.merge(validateMavenPom(repository.getMavenPom()));
+        report.merge(validateFileExists(this.repository, MAVEN_WORKFLOW_PATH, "Workflow for a Maven release."));
```

Comment:
remove this parameter as we have it already as instance variable
```suggestion
        report.merge(validateFileExists(MAVEN_WORKFLOW_PATH, "Workflow for a Maven release."));
```

Reply:
The same here